action/protocol/poll/staking_committee.go

```diff
@@ -31,13 +31,15 @@ func NewStakingCommittee(
 	gs Protocol,
 	cm protocol.ChainManager,
 	getTipBlockTime GetTipBlockTime,
+	getEpochHeight GetEpochHeight,
+	getEpochNum GetEpochNum,
 	staking string,
 	scoreThreshold *big.Int,
 ) (Protocol, error) {
 	var ns *NativeStaking
 	if staking != "" {
 		var err error
-		if ns, err = NewNativeStaking(cm, getTipBlockTime, staking); err != nil {
+		if ns, err = NewNativeStaking(cm, getTipBlockTime, getEpochHeight, getEpochNum, staking); err != nil {
```

Comment:
should we keep getEpochHeight and getEpochNum in staking_committee? They are not used in native staking.

Reply:
fixed in latest push